CMakeLists.txt

```diff
@@ -23,12 +25,14 @@ add_custom_command(
 add_custom_target(coordinatorlib
   ertgo build ${TRIMPATH} -buildmode=c-archive -tags enclave
   -o libcoordinator.a
+  -ldflags "-X 'main.Version=${PROJECT_VERSION}' -X 'main.BuildDate=${date}'"
```

Comment:
I informed myself about Project Oak at the weekend. They took some effort to achieve reproducible builds. I think this would be desirable for marblerun in the long term, too. Maybe we should drop build date. Maybe use git commit hash instead. What's your opinion? (Also @m1ghtym0)

Reply:
git commit hash sounds good, probably gives a better indication of when and what was built